scripts/workspace-status.sh

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-echo "STABLE_GIT_COMMIT $(git rev-parse HEAD)"
+if [ -z "$(git status --porcelain)" ]; then
+  echo "STABLE_GIT_COMMIT $(git rev-parse HEAD)"
```

Comment:
Just to confirm: if the key is not output, its value in `x_defs` will be empty string?

Reply:
Yup!